config/grafana/dashboards/release_dashboard.json

```diff
@@ -606,16 +606,13 @@
           },
           "editorMode": "code",
           "exemplar": false,
-          "expr": "sum by (namespace) (release_total{deployment_reason=\"Failed\"})",
           "instant": true,
           "interval": "",
           "legendFormat": "__auto",
           "range": false,
           "refId": "A"
         }
-      ],
-      "title": "Failed deployments",
```

Comment:
Shouldn't this entire panel entry be removed? Not just the title, type, and expr of it